### PR TITLE
Fix spelling mistakes & grammar

### PR DESCRIPTION
**Pull request by petersmythe**
<Include a few sentences describing the overall goals for this Pull Request. Please do not remove the checklist below, pull requests missing it will be ignored.>

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: doc/en/api/1.0.0/coverages.yaml [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ definitions:
         description: The number of decimal places to use when encoding floating point numbers from data of this feature type.
       responseSRS:
         type: object
-        description: The srs's that the WFS service will advertise in the capabilities document for this feature type (overriding the global WFS settings).
+        description: The Ss that the WFS service will advertise in the capabilities document for this feature type (overriding the global WFS settings).
```
Comment (mprins) [inline doc/en/api/1.0.0/coverages.yaml:1021]: this seems incorrect, should be SRSs
```suggestion
        description: The SRSs that the WFS service will advertise in the capabilities document for this feature type (overriding the global WFS settings).
```

**Review by mprins — CHANGES_REQUESTED** [review]
A small typo snuck in

**Review by mprins — APPROVED** [review]

**Review by bradh — APPROVED** [review]
Added a couple of cleanups

**Comment by petersmythe** [comment]
Yes, weird GitHub editor strangeness, that I tried to fix with ec4b2998452ec43dc49290dff44611e5f89cb9ce.  Thank you for reviewing and correcting my edits.

**Review by mprins — APPROVED** [review]
LGTM

**Comment by mprins** [comment]
windows build is failing to resolve some dependencies (not from the osgeo repo) which are unrelated to these changes

**Pull request merged**